soccer/SystemState.hpp

```diff
@@ -70,9 +70,14 @@ class SystemState {
      */
 
     /** @ingroup drawing_functions */
-    void drawLine(const Geometry2d::Line& line, const QColor& color = Qt::black,
+    void drawLine(const Geometry2d::Segment& line,
```

Comment:
This is mostly backwards compatibility problems. and meh. I think that drawLine should draw an actually Line as instead of just a line segment, but thats not the behavior expected right now so idk.